openghg/standardise/_standardise.py

```diff
@@ -1,18 +1,38 @@
 from pathlib import Path
-from typing import Dict, Literal, Optional, Union
+from typing import Dict, Literal, Optional, Union, Any
 from pandas import Timedelta
 
+from openghg.store.base import get_data_class
 from openghg.cloud import create_file_package, create_post_dict
 from openghg.objectstore import get_writable_bucket
 from openghg.util import running_on_hub
 from openghg.types import optionalPathType, multiPathType
 
 
+def standardise(bucket: str, data_type: str, filepath: multiPathType, **kwargs: Any) -> dict:
+    """Generic standardise function, used by data-type specific versions.
+
+    args:
+        bucket: object store bucket to use
+        data_type: type of data to standardise
+        filepath: path to file(s) to standardise
+        **kwargs: data type specific arguments, see specific implementations below.
+
+    returns:
+        Dictionary of result data.
+    """
+    dclass = get_data_class(data_type)
+    with dclass(bucket) as dc:
+        result = dc.read_file(filepath=filepath, **kwargs)
+    return result
+
+
 def standardise_surface(
-    filepaths: multiPathType,
     source_format: str,
     network: str,
     site: str,
+    filepath: Optional[multiPathType] = None,
```

Comment:
Okay, I think that should be easy enough to do. I guess these changes were meant to be somewhat temporary, until I actually update `BaseStore`. I'm just trying to limit explicit dependencies on how `BaseStore` works to a smaller number of places, like the `standardise` module. But I think I can just do like you say and throw a `NotImplementedError` for the data classes that can't accept a list.

Reply:
I think I misunderstood what you meant. I have both here because some of the docs and other parts of the code use 'filepaths' and it is possible that some people have scripts that use 'filepaths', so I didn't want to make a breaking change. I can add a deprecation warning and change all instances of 'filepaths' to 'filepath' in OpenGHG and the tutorials.

Reply:
The individual `standardise_*` functions have 'filepath' types that match the corresponding `read_file` functions, which are all included in `multiPathType`, so this part of the `standardise_*` functions hasn't really changed, except for `standardise_surface`, which I needed to have a `filepath` argument to let it depend on `standardise`.

Maybe it would be good to add that error checking, but I don't want to do it in this PR, since I haven't really changed this aspect of how the code works.